windows/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -107,6 +107,9 @@
 - name: "Wait for VM network adapter is connected"
   include_tasks: ../../common/vm_wait_network_connected.yml
 
+- name: "Get VM's primary network adapter MAC address after poweron VM"
+  include_tasks: ../../common/vm_wait_primary_nic_mac.yml
```

Comment:
Missing `when: vm_primary_nic_mac is undefined or not vm_primary_nic_mac`

Reply:
Please also move vm_get_vm_info.yml after power-on in `windows/deploy_vm/deploy_vm_from_iso.yml`

Reply:
Done. Thanks